Production_scripts/Process_extracts/Produce_extract_tests/extract_nrs_tests.R

```diff
@@ -0,0 +1,38 @@
+####################################################
+# Name of file - extract_nrs_tests.R
+# Original Authors - Bateman McBride
+# Original Date - July 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               acute processed file.
+#####################################################
+
+# Load packages
+library(createslf)
+
+# Read in Data-----------------------------------------
+
+year <- "1920"
+
+# Read new data file
+new_data <- readr::read_rds(get_source_extract_path(year, "Deaths", ext = "rds"))
```

Comment:
Permissions are preventing me from opening the rds file, but i tested the script using the zsav file. I think permissions were sorted, maybe worth running the nrs deaths script again to create a new file